styles/product-slider.scss

```diff
@@ -35,12 +35,19 @@
 
   .splide__arrows {
     position: absolute;
-    transform: translate(128px, -73.5px);
-
-    @media screen and (max-width: 845px) {
-      transform: translate(44px, 25px);
-      left: 50%; 
-      bottom: 0;
+    transform: translate(44px, 25px);
+    left: 50%;
+    bottom: 0;
+
+    @media screen and (min-width: 845px) {
+      transform: unset;
+      bottom: unset;
+      top: -73px;
+      left: 127px;
+
+      [dir='ltr'] & {
+        left: calc(100% - 24px);
+      }
```

Comment:
still need a small alignment for the slider arrows
<img width="898" alt="Screen Shot 2023-04-06 at 15 19 49" src="https://user-images.githubusercontent.com/108662497/230424390-b22d940a-a72b-49c4-9a62-f49e6466089d.png">